packages/components-react/package.json

```diff
@@ -70,6 +70,6 @@
     "@gemeente-denhaag/typography": "workspace:*"
   },
   "peerDependencies": {
-    "react": "^18.0.0"
+    "react": "^18.3.1"
```

Comment:
Als je deze aanpast krijgen gebruikers met 18.2.0 foutmeldingen, terwijl het misschien wel gewoon werkt. Is dat de bedoeling?

Reply:
Goed punt! Ik had eerst issues in storybook na het updaten van de React dependencies. Maar heb het nog een keer geprobeerd met de oude peerdependencies, en dat lijkt gewoon goed te gaan. Dus zal dat even committen. :)